modules/nw-sriov-create-object.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/using-dpdk-and-rdma.adoc
+
+:_content-type: PROCEDURE
+[id="nw-sriov-create-object_{context}"]
+
```

Comment:
Please remove this new line so that the ID is directly above the heading.

Reply:
Done.